watchdog/canister/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "watchdog"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+# [lib]
+# path = "lib.rs"
+# crate-type = ["cdylib"]
+
+[[bin]]
+name = "watchdog"
+path = "src/main.rs"
+
+[dependencies]
+candid = "0.8.2"
+ic-cdk = "0.6.0"
+ic-cdk-macros = "0.6.0"
+ic-cdk-timers = "0.1"
+serde = { version = "1.0.158", features = [ "derive" ] }
+serde_json = "1.0.94"
+hex = "0.4.3"
+async-trait = "0.1.67"
+regex = "1.7.0"
+futures = "0.3.27"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+tokio = { version = "1.15.0", features = [ "full" ] }
```

Comment:
This is a leftover from `ic-http-mock`, first implementation was not as a separate crate, but rather a mod inside watchdog canister. It was conditionally adding `tokio` to `.dependencies` because it's using non-blocking `sleep` from `tokio` to mock a delayed response. Unfortunately `tokio` is not supported for `wasm32`. 

Thank you for catching this. Let me clean this up.

Reply:
Done.